File: 0001_0100/0044/0044.py
with open("input.txt", "r") as f, open("output.txt", "w") as q:
    s = f.read()
    k = 0
    for x in (">>-->", "<--<<"):
        n = s.find(x, 0)
        while n > -1:
            k += 1
            n = s.find(x, n + 1)
    q.write(str(k))

File: 0001_0100/0044/0044.go
package main

import (
	"fmt"
	"os"
)

func main() {
	f, _ := os.Open("input.txt")
	q, _ := os.Create("output.txt")
	var s string
	fmt.Fscan(f, &s)
	n := 0
	for i := 0; i < len(s)-4; i++ {
		if s[i:][:5] == ">>-->" || s[i:][:5] == "<--<<" {
			n++
		}
	}
	fmt.Fprint(q, n)
}
